code/datums/mutations/_mutations.dm

```diff
@@ -90,6 +90,11 @@
 /datum/mutation/human/proc/on_acquiring(mob/living/carbon/human/acquirer)
 	if(!acquirer || !istype(acquirer) || acquirer.stat == DEAD || (src in acquirer.dna.mutations))
 		return TRUE
+	// MONKESTATION ADDITION START -- CORTICAL_BORERS
+	if(acquirer.has_borer())
```

Comment:
not a change request, just curious as to why this is a thing

Reply:
partially because H.A.R.S.

Reply:
wouldn't it be better to just have this check on the specific troublesome mutations, then?